ServerCore/Startup.cs

```diff
@@ -204,9 +215,15 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseAuthentication();
             app.UseAuthorization();
 
+            app.UseAntiforgery();
```

Comment:
This is another example of a line where I have no idea whether it's a WebAssembly usage thing, a .net 8.0 thing, or something completely different.

Reply:
Refactored .net 8.0 out